src/components/SliderButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { sliderImage } from "./SliderImageWallet";
+
+type SliderImageWalletProps = {
+  sliderValue: number;
+  setSliderValue: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export default function WalletImageSlider({
+  sliderValue,
+  setSliderValue,
+}: SliderImageWalletProps) {
+  let textValue: string;
+  if (sliderValue == 11) {
+    textValue = "Plus Cash";
+  } else if (sliderValue == 1) {
+    textValue = "1 Card";
+  } else {
+    textValue = `${sliderValue} Cards`;
+  }
+
+  return (
+    <div className="flex flex-col justify-center w-full z-40 absolute bottom-0">
+      <div className="relative">
+        <input
+          type="range"
+          className="w-full absolute
+            appearance-none h-0 -bottom-2 outline-none z-10"
+          min={0}
+          max={sliderImage.length - 1}
+          step={1}
+          value={sliderValue}
+          onChange={(event) => {
+            setSliderValue(event.target.value);
+          }}
```

Comment:
You can refactor this, event automatically gets passed into setSliderValue if you call this 
```suggestion
          onChange={setSliderValue}
```

Reply:
this appears to work for WalletImageSlider Input tag, but causes issues when refactoring the same for SliderButton.

Reply:
What are the issues? Have you tried to fix them?

Reply:
I created a handler that takes in the event and uses it to set the state and pass it onChange.